hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -555,7 +555,7 @@ def title(self):
         if not isinstance(title, str):
             # We only want a string, if this is not the case, set to default
             _logger.info("Title can't be imported.")
-            title = ''
+            title = os.path.splitext(os.path.split(self.filename)[1])[0]
```

Comment:
I don't think that we need to do this. HyperSpy stores the filename in ``s.tmp_metadata.filename`` and uses it for the signal title if ``metadata.General.title`` is not defined.

Reply:
Indeed, I thought this was the case, but it doesn't work. I would expect this to be happen in [`load_with_reader`](https://github.com/hyperspy/hyperspy/blob/RELEASE_next_patch/hyperspy/io.py#L298-L308), but this is actually not!

Reply:
I've just tested it with a dm3 file and it works for me.

Reply:
Yes, shall stick to hyperspy default (empty string) as for new data, then?
```python
>>> s = hs.signals.Signal1D([0,1])
>>> s.metadata
├── General
│   └── title = 
└── Signal
    ├── binned = False
    └── signal_type = 
```

Reply:
Sounds good.

Reply:
Done, reset to previous commit.